hw2/03-count.js

```diff
@@ -1,3 +1,18 @@
-// Add your code here
+const input = document.getElementById("userInput");
+const paragraph = document.getElementById("textContainer").innerHTML;
 
-input.addEventListener('keydown', handleKeyDown);
+input.addEventListener("keydown", handleKeyDown);
```

Comment:
here, rather than using keydown, may be you can use input because it is more suitable to reach changes and update the hightlighted text.
(I also used keydown and have learnt it now that input is more responsiveness)

Reply:
I agree that the input is more responsive than input, but professor was talking about keydown in class because it simulates pressing enter or space after typing your search query. I think the homework doc also implied that a keydown should be used due to the linking of the page for it

Reply:
Yes, I agree to that too. "Input" type is more responsive to the immediate inputs given, instead of "keydown". But like Kobe said, since the homework specified "keydown" with this particular exercise, I have used the same.